src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -196,23 +196,18 @@ private HashMap<String, Integer> saveDryRunStructures(HashMap<String, CmpdRegMol
 	}
 
 	@Override
+	@Transactional
```

Comment:
Does this parent function need to be Transactional? Just asking based on the pattern that `registerSDF` is not Transactional while the inner function is.